src/modules/dcc/DccFileTransfer.cpp

```diff
@@ -366,8 +365,8 @@ void DccRecvThread::run()
 						// Readed something useful...write back
 						if(((uint)(readLen + m_pFile->pos())) > m_pOpt->uTotalFileSize)
 						{
-							postMessageEvent(__tr_no_lookup_ctx("WARNING: The peer is sending garbage data past the end of the file","dcc"));
-							postMessageEvent(__tr_no_lookup_ctx("WARNING: Ignoring data past the declared end of file and closing the connection","dcc"));
+							postMessageEvent(__tr_no_lookup_ctx("WARNING: the peer is sending garbage data past the end of the file","dcc"));
```

Comment:
The reason is simple, there are another 500 of these not capitalized so instead of changing 500 I chose to change 10 or 20.these instances are made to match those other majority.